### PR TITLE
DisableSharingForNonOwnersStatus Change

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ Accept wildcard characters: False
 This parameter prevents non-owners from sharing.
 
 > [!NOTE]
-> This parameter is available only in SharePoint Online Management Shell Version 16.0.4613.1211 or later.  
+> This parameter is available only in SharePoint Online Management Shell Version 16.0.4613.1211 or later. DisableSharingForNonOwnersStatus is not a persisted setting but rather an analysis of the state of the site collection. The purpose of this is to get this setting, and it’s not guaranteed that other settings returned are correct. To get other settings and values, please use the Get-SPOSite without this parameter to ensure everything is displayed correctly.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:209]:
```suggestion
> This parameter is available only in SharePoint Online Management Shell Version 16.0.4613.1211 or later. DisableSharingForNonOwnersStatus is not a persisted setting but rather an analysis of the state of the site collection. The purpose of this is to get this setting, and it’s not guaranteed that other settings returned are correct. To get other settings and values, use the Get-SPOSite without this parameter to ensure everything is displayed correctly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @salarson, kindly commit the minor suggested change. Many thanks.

**Review by dariomws — APPROVED** [review]
Approve but I think we can move this outside the note. In addition, you should not use note format (see -Filter parameter).

**Review by get-itips — APPROVED** [review]
agree with @dariomws

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**